x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/FunctionInfo.java

```diff
@@ -18,6 +18,15 @@
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.CONSTRUCTOR)
 public @interface FunctionInfo {
+    /**
+     * If this function implements an operator, what is its symbol?
+     * <p>
+     *     This exists entirely to add to the Kibana function definition
+     *     json files.
+     * </p>
+     */
+    String operator() default "";
```

Comment:
I'd like to stick *kibana's* definition of an operator here - it's important that what we stick in these fields is compatible with that. And I had a conversation this morning where we could come up with three equally useful definitions of operator....